src/js/Content/Features/Community/Inventory/FInventoryMarketHelper.js

```diff
@@ -17,10 +17,10 @@ export default class FInventoryMarketHelper extends Feature {
                 if (!g_ActiveInventory.selectedItem.description.market_hash_name) {
                     g_ActiveInventory.selectedItem.description.market_hash_name = g_ActiveInventory.selectedItem.description.name;
                 }
-                let market_expired = false;
-                if (g_ActiveInventory.selectedItem.description) {
-                    market_expired = g_ActiveInventory.selectedItem.description.descriptions.reduce(
-                        (acc, el) => (acc || el.value === "This item can no longer be bought or sold on the Community Market."),
+                let marketRestriction = false;
+                if (g_ActiveInventory.selectedItem.description.owner_descriptions) {
+                    marketRestriction = g_ActiveInventory.selectedItem.description.owner_descriptions.reduce(
+                        (acc, el) => (acc || /\[date\]\d+\[\/date\]/.test(el.value)),
```

Comment:
Ideally this should test the whole message, but it is localized...so this may break some items that have dates in the owner description.